### PR TITLE
chore: Support OPALA or envoy-opa for authentication (MAPCO-3852)

### PR DESCRIPTION
**Pull request by RonitKissis**
Support authentication via both Opala and opa-envoy through values configuration

### CODE REVIEW

File: helm/raster/config/log_format.conf [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            '"http.x_forwarded_for": "$http_x_forwarded_for",'
+            '"http.user_agent": "$http_user_agent",'
+            '"http.request_time": $request_time,'
+            '"request_length": "$request_length",'
+            '"http.host": "$host",'
+            '"upstream_response_time": $upstream_response_time,'
+            '"body_bytes_sent": "$body_bytes_sent",'
+            '"remote_addr": "$remote_addr",'
+            '"remote_user": "$remote_user",'
+            '"connection":"$connection",'
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply helm/raster/config/log_format.conf:24]: why? this is the same log format that existed in pycsw but i just transfered this code to a separate file

[the remark this answers is not in the document]
Comment (asafMasa) [reply helm/raster/config/log_format.conf:24]: please edit like in mapproxy

File: helm/raster/values.yaml [hunk 2]
```diff
@@ -1,30 +1,27 @@
 global:
   cloudProvider:
-    dockerRegistryUrl: my-registry-url.io
-    imagePullSecretName: "my-registry-secret"
+    dockerRegistryUrl: acrarolibotnonprod.azurecr.io
+    imagePullSecretName: acr-registry
```
Comment (CL-SHLOMIKONCHA) [inline helm/raster/values.yaml:4]: place an empty string

File: helm/raster/values.yaml [hunk 3]
```diff
@@ -38,8 +35,8 @@ splunkIndex: ""
 imagePullPolicy: Always
 
 cloudProvider:
-  dockerRegistryUrl: my-registry-url.io
-  imagePullSecretName: "my-registry-secret"
+  dockerRegistryUrl: acrarolibotnonprod.azurecr.io
```
Comment (CL-SHLOMIKONCHA) [inline helm/raster/values.yaml:38]: place an empty string

File: helm/raster/values.yaml [hunk 4]
```diff
@@ -38,8 +35,8 @@ splunkIndex: ""
...
+  imagePullSecretName: acr-registry
```
Comment (CL-SHLOMIKONCHA) [inline helm/raster/values.yaml:39]: place an empty string

**Review by CL-SHLOMIKONCHA — CHANGES_REQUESTED** [review]

File: helm/raster/config/log_format.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (RonitKissis) [reply helm/raster/config/log_format.conf:37]: done

File: helm/raster/values.yaml [hunk 5]
```diff
@@ -1,30 +1,27 @@
 global:
   cloudProvider:
-    dockerRegistryUrl: my-registry-url.io
-    imagePullSecretName: "my-registry-secret"
+    dockerRegistryUrl: acrarolibotnonprod.azurecr.io
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply helm/raster/values.yaml]: done

File: helm/raster/values.yaml (hunk 3 above)
Comment (RonitKissis) [reply helm/raster/values.yaml]: done

**Review by asafMasa — APPROVED** [review]

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

**Pull request merged**